package.json

```diff
@@ -13,9 +13,7 @@
     "coverage": "istanbul cover -x test.js _mocha",
     "test": "mocha"
   },
-  "files": [
```

Comment:
Please keep it and add `index.d.ts` here.

Reply:
Ok assume you want me to remove the .npmignore then? It is doing the dam thing

Reply:
Yes, it's easier to whitelist files instead of blacklisting them in this case.

Reply:
Sorry they should have been same thing* on my phone. Ok will do